basicMysql/login/login.php

```diff
@@ -0,0 +1,69 @@
+<?php
+  session_start();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>User-Login</title>
+  <style>
+    <?php include "../../style.css" ?>
+  </style>
+</head>
+<body>
+  <?php
+    if(isset($_SESSION['login_user'])){
+      header("location: ../../index.php");
+    }
+
+    //require("rootClass.php");
+    //$loginObj = new RootClass();
+  ?>
+
+  <h1>Login-Page</h1>
+  <form action="logged-in.php" method="post">
+    <dl>
+      <dt><label for="username">Enter User-Name</label></dt>
+      <dd>
+        <input type="text" name="username" placeholder="Enter user-name" value="<?php if(isset($_SESSION['logged_user'])){echo $_SESSION['logged_user'];} ?>" required>
+        <span>
+          <?php
+            if(isset($_SESSION['logged_nstatus']) && $_SESSION['logged_nstatus']){
```

Comment:
Block of PHP code can be written in a separate file only single-line PHP code is preferred.